### PR TITLE
Display winning line

### PR DESCRIPTION
**Pull request by Chroq**
## Description

In this PR, we display the winning line.

image

### CODE REVIEW

File: engine/game.test.js [hunk 1]
```diff
@@ -1,23 +1,23 @@
 var dijkstra = require("../node_modules/dijkstrajs/dijkstra.js");
 var find_path = dijkstra.find_path;
 
-import { isWon } from "./game.js";
+import { isWon, getWinningPath } from "./game.js";
```
Comment (JulienMattiussi) [inline engine/game.test.js:4]: iswon is obsolete here

File: components/boards/Playboard.js [hunk 2]
```diff
@@ -34,8 +34,21 @@ function Playboard(props) {
...
 
-    if (isWon(updatedGrid, player)) {
+    const winningPath = getWinningPath(updatedGrid, player);
+
+    console.log("Winning Path : ", winningPath);
+    if (winningPath) {
       setWinner(player);
+
+      const winningGrid = grid.map(function(value, index) {
+        if (_.indexOf(winningPath, (index + 1).toString(10)) >= 0) {
```
Comment (JulienMattiussi) [inline components/boards/Playboard.js:44]: maybe have a hexagonIsInPath would have been more clear here

File: engine/game.test.js (hunk 1 above)
Comment (Chroq) [reply engine/game.test.js]: Fixed, TY

File: components/boards/Playboard.js (hunk 2 above)
Comment (JulienMattiussi) [reply components/boards/Playboard.js:43]: (and unit-testable)

Comment (Chroq) [reply components/boards/Playboard.js:43]: Ok, I extract this part into a function

Comment (Chroq) [reply components/boards/Playboard.js]: Fixed

**Pull request merged**